cli/command/container/rm.go

```diff
@@ -52,22 +52,22 @@ func runRm(dockerCli *command.DockerCli, opts *rmOptions) error {
 	}
 
 	errChan := parallelOperation(ctx, opts.containers, func(ctx context.Context, container string) error {
+		container = strings.Trim(container, "/")
```

Comment:
this is the only "functional" change; I considered a container name containing only slashes to be "empty", but happy to change this back :smile: